### PR TITLE
Editorial pass, part 1

### PR DESCRIPTION
**Pull request by janaiyengar**
Minor nits and changes.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -128,12 +128,12 @@ QUIC establishes a connection, which is a stateful interaction between a client
 and server. The primary purpose of a connection is to support the structured
 exchange of data by an application protocol.
 
-Streams are means by which an application protocol exchanges information.
-Streams are ordered sequences of bytes. Two types of stream can be created:
-bidirectional streams, which allow both endpoints to send data; and
-unidirectional streams, which allow a single endpoint to send. A credit-based
-scheme is used to limit stream creation and to bound the amount of data that
-can be sent.
+Within a connection, streams are the means by which an application protocol
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:131]: I don't like this change much.

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -425,7 +425,7 @@ by the application to determine how to allocate resources to active streams.
 
 This document does not define an API for QUIC, but instead defines a set of
 functions on streams that application protocols can rely upon.  An application
-protocol can assume that an implementation of QUIC provides an interface that
+protocol can assume that an QUIC implementation provides an interface that
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:428]:
```suggestion
protocol can assume that a QUIC implementation provides an interface that
```

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -800,17 +800,18 @@ enable a receiver to limit memory commitment to a connection and to apply back
...
 
 Data sent in CRYPTO frames is not flow controlled in the same way as stream
 data.  QUIC relies on the cryptographic protocol implementation to avoid
-excessive buffering of data; see {{QUIC-TLS}}.  The implementation SHOULD
-provide an interface to QUIC to tell it about its buffering limits so that there
-is not excessive buffering at multiple layers.
+excessive buffering of data; see {{QUIC-TLS}}. To avoid excessive buffering at
+multiple layers, QUIC implementations SHOULD provide an interface for the
+cryptographic protocol implementation to communicate its buffering limits to the
+QUIC implementation.
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:814]:
```suggestion
cryptographic protocol implementation to communicate its buffering limits.
```

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -976,10 +976,11 @@ Once a receiver advertises a stream limit using the MAX_STREAMS frame,
 advertising a smaller limit has no effect.  A receiver MUST ignore any
 MAX_STREAMS frame that does not increase the stream limit.
 
-As with stream and connection flow control, this document leaves when and how
-many streams to advertise to a peer via MAX_STREAMS to implementations.
-Implementations might choose to increase limits as streams close to keep the
-number of streams available to peers roughly consistent.
+As with stream and connection flow control, this document leaves to
+implementations the decisions of when and how many streams should be advertised
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:980]:
```suggestion
As with stream and connection flow control, this document leaves implementations
to decide when and how many streams should be advertised
```

**Review by martinthomson — APPROVED** [review]
A few nits and quibbles, but that is where we are.

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (janaiyengar) [reply draft-ietf-quic-transport.md:131]: The paragraph above talks about connection, and the following paragraph talks about setting one up. Streams are brought up here first but their relationship to connections is not clear, even after reading this paragraph. I wanted some glue.

File: draft-ietf-quic-transport.md [hunk 5]
```diff
@@ -128,12 +128,12 @@ QUIC establishes a connection, which is a stateful interaction between a client
...
 exchange of data by an application protocol.
 
-Streams are means by which an application protocol exchanges information.
-Streams are ordered sequences of bytes. Two types of stream can be created:
-bidirectional streams, which allow both endpoints to send data; and
-unidirectional streams, which allow a single endpoint to send. A credit-based
-scheme is used to limit stream creation and to bound the amount of data that
-can be sent.
+Within a connection, streams are the means by which an application protocol
+exchanges information.  Streams are ordered sequences of bytes. Two types of
```
Comment (ianswett) [inline draft-ietf-quic-transport.md]:
```suggestion
Application protocols exchange information over a QUIC connection via streams, which
are ordered sequences of bytes. Two types of
```

**Pull request merged**